### PR TITLE
[13.0] Finnish localization features without CoA

### PR DESCRIPTION
**Pull request by mlaitinen**
This commit includes the following features to the Finnish localization:

- Business code (unique identifier of a company) for `res.partner`
- eInvoicing address and eInvoicing operator for `res.partner` and `res.company`
- Finnish payment reference types

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mlaitinen** [comment]
ping @AdrienHorgnies and @jco-odoo

**Comment by AdrienHorgnies** [comment]
@mlaitinen Saw the ping. Will come back on it tomorrow

File: addons/l10n_fi/models/res_partner_operator_einvoice.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    ]
+
+    def name_get(self):
+        """
+        Overwrite core method to add value of `identifier` ("Identifier")
+        field into name of records.
+        """
+        result = []
+        for operator in self:
+            name = " - ".join([operator.identifier, operator.name])
```
[the remark this answers is not in the document]
Comment (AdrienHorgnies) [reply addons/l10n_fi/models/res_partner_operator_einvoice.py]: (It's just for the reference, I'll cherry pick and change it myself anyway)

[the remark this answers is not in the document]
Comment (mlaitinen) [reply addons/l10n_fi/models/res_partner_operator_einvoice.py]: I believe this part of the code has originally been developed targeting an Odoo version that didn't support Python 3.7.

**Comment by jarmokortetjarvi** [comment]
What is the status on this? I'm guessing it's not plausible to get this merged to 13.0, but is there any reason the same PR couldn't be merged on master? I'm trying to decide if this functionality should be ported in OCA modules.

It would be absolutely wonderful for all Finnish users if these additions could be added to Odoo.

**Comment by jco-odoo** [comment]
@jarmokortetjarvi @mlaitinen There is code for the references in standard already.  The problem we have with the e-invoice operators is that if we add them like this, it does not make a lot of sense as they are not really used somewhere.  If we got a more or less standard export (and maybe even import) of the FInvoice with a qweb template that would already make a big difference.

**Pull request closed without merging**